src/node/db/PadManager.js

```diff
@@ -79,27 +80,25 @@ let padList = {
     await this.load();
 
     if (!this.sorted) {
-      this.list.sort();
+      this.cachedList = Array.from(this.list).sort();
```

Comment:
In the previous version ```this.sorted``` was used to keep whether ```this.list``` (then a list, now a set) was sorted.

What is its meaning now?
I am asking because this could be a semantic change, and thus introduce subtle bugs down the road.

My proposal is to further analyze its meaning, and eventually keep the invariant (sorted means that something is sorted), or get rid of it altogether.

Reply:
I'm not sure to understand. The `sorted` variable was only used in the pad list structure to know if `getPads` needs or not to sort the list before returning the array. I think it was just for performance, and not used anywhere else.

Reply:
I asked because when reviewing I can only look from an high level, without digging deep as you have done for proposing this.

So, `sorted` is probably not used. Makes sense trying to get rid of it, isn't it?

Reply:
I think I understood. I pushed another version that does not use the `sorted` variable.